lib/fhmongodb.js

```diff
@@ -477,4 +496,54 @@ Database.prototype.collection = function (name, cb) {
   this.db.collection(name, cb);
 };
 
+// Same as `collection` but returns a collection instance if no callback is
+// passed and also accepts extra options
+Database.prototype.getCollectionInstance = function (name, options, callback) {
+  if (!this.db) {
+    if (callback) {
+      return callback(new Error("no database open"));
+    }
+    throw new Error("no database open");
+  }
+
+  return this.db.collection(name, options, callback);
+};
+
+Database.prototype.renameCollection = function (fromCollection, toCollection, options, callback) {
+  if (!this.db) {
+    if (callback) {
+      return callback(new Error("no database open"));
+    }
+    throw new Error("no database open");
+  }
+
+  return this.db.renameCollection(fromCollection, toCollection, options, callback);
+};
+
+// Raw MongoDB `listCollections` returning a CommandCursor like object
+Database.prototype.listCollections = function (filter, options) {
+  if (!this.db) {
+    throw new Error("no database open");
+  }
+
+  var cursor = this.db.listCollections(filter, options);
+  var databaseName = this.db.databaseName;
+
+  // MongoDB `CommandCursor` like object
+  function CommandCursor(cursor) {
+    this.toArray = function (callback) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)